src/CommunicationHandlerApi/Controllers/WhatsAppController.cs

```diff
@@ -31,21 +31,27 @@ public WhatsAppController(IWhatsappHandlerService whatsappHandlerService)
             _whatsappHandlerService = whatsappHandlerService;
         }
 
+        [HttpPost(Name = "Whatsapp_Ping")]
+        public async Task<IActionResult> Get()
```

Comment:
Just curious - Was this one lacking or just overlooked? https://github.com/CoditEU/htc-social-connect/blob/master/src/CommunicationHandlerApi/Controllers/HealthController.cs#L38

Just to see if we can improve the template

Reply:
just debugging purposes, tom.  manual deploy works, but my automated deployment through azure pipelines results in the app being unavailable